packages/core/content-manager/admin/src/pages/EditView/components/FormInputs/Relations/Relations.tsx

```diff
@@ -118,6 +120,8 @@ interface Relation extends Pick<RelationResult, 'documentId' | 'id' | 'locale' |
     id: RelationResult['id'];
     locale?: RelationResult['locale'];
     position: RelationPosition;
+    // Added this property to prevent call useDocument with a not temporary relation (one already saved in the database)
+    temporary?: boolean;
```

Comment:
probably the apiData was enough to discriminate between a new relation connected (and not save) and one already in memory but I wanted to be sure so I added this boolean prop